lib/mayaUsd/nodes/proxyShapePlugin.cpp

```diff
@@ -48,9 +48,9 @@ MString _registrantPluginName;
 bool _useVP2RenderDelegate = false;
 
 TF_DEFINE_ENV_SETTING(
-    VP2_RENDER_DELEGATE_PROXY,
+    MAYAUSD_DISABLE_VP2_RENDER_DELEGATE,
     false,
-    "Switch proxy shape rendering to VP2 render delegate.");
+    "Disable VP2RenderDelegate from handling rendering of the proxy shape. Switch to legacy mode");
```

Comment:
I'm all for "namespacing" the env setting by prepending the `MAYAUSD_` prefix, so no issue with the name of this variable being changed.

I do have a mild aversion though to options that are of the "enable this setting to disable this feature/behavior" variety. I feel like they often involve some additional mental logic gymnastics to deduce the meaning when they're used as below in the form of "if not disabled then use the new thing". Is that preferable to an inverse setting like this?:

```C++
TF_DEFINE_ENV_SETTING(
    MAYAUSD_ENABLE_VP2_RENDER_DELEGATE,
    true,
    "Use the VP2 render delegate for proxy shape rendering. Disable to switch to legacy mode.");
```

Contentious though it may be, this follows the example of `MAYA_ENABLE_LEGACY_VIEWPORT`, which is admittedly disabled by default, but it's a little more straightforward to reason about. Turn the option off to disable the feature, and turn it on (or leave it alone) to enable it. To my eye, `MAYAUSD_ENABLE_VP2_RENDER_DELEGATE=0` a bit more clearly suggests that you want the VP2 render delegate **disabled**.